control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1453,6 +1453,12 @@ paths:
           required: false
           schema:
             type: boolean
+      requestBody:
```

Comment:
Since we're adding a body now, maybe we can move the `republish` and `reuse_existing` to the body as well?

Reply:
Yep. We can, that would be good.